lib/livebook/hubs/team_client.ex

```diff
@@ -519,6 +572,15 @@ defmodule Livebook.Hubs.TeamClient do
     end
   end
 
+  defp build_authorization_groups(%{authorization_groups: authorization_groups}) do
+    for authorization_group <- authorization_groups do
+      %Teams.AuthorizationGroup{
+        oidc_provider_id: authorization_group.oidc_provider_id,
```

Comment:
Let's call it `provider_id` both on server and here, this will make it easier if somehow we need to support authorization groups from other providers in the future. :)